### PR TITLE
Address issue #495

### PR DESCRIPTION
**Pull request by shigeya**
Described two possible cases where 1) there is no update, and
2) an update performed within less than one second after creation.

Part of the text by Amy Guy. thanks.

***
<a href="https://pr-preview.s3.amazonaws.com/shigeya/did-core/pull/540.html" title="Last updated on Jan 20, 2021, 12:01 AM UTC (72e36d7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/540/c8bc6b1...shigeya:72e36d7.html" title="Last updated on Jan 20, 2021, 12:01 AM UTC (72e36d7)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3858,7 +3858,11 @@ <h4>updated</h4>
 DID document metadata SHOULD include an <code>updated</code> property to
 indicate the timestamp of the last <a href="#update">Update operation</a> for
 the document version which was resolved. The value of the property MUST follow
-the same formatting rules as the <code>created</code> property.
+the same formatting rules as the <code>created</code> property. The updated property
+is omitted if an Update operation has never been performed on the DID document.
+If an <code>updated</code> property exists, it may be the same value as the
```
Comment (rhiaro) [inline index.html:3863]:
```suggestion
If an <code>updated</code> property exists, it can be the same value as the
```

**Review by rhiaro — APPROVED** [review]
My only suggestion is to change "may" to "can" to avoid any perception of normative language. Otherwise, looks good to me!

**Review by msporny — APPROVED** [review]

**Comment by shigeya** [comment]
Thanks.

**Review by peacekeeper — APPROVED** [review]
Looks good, thanks for the contribution!

File: index.html [hunk 2]
```diff
@@ -3858,7 +3858,11 @@ <h4>updated</h4>
 DID document metadata SHOULD include an <code>updated</code> property to
 indicate the timestamp of the last <a href="#update">Update operation</a> for
 the document version which was resolved. The value of the property MUST follow
-the same formatting rules as the <code>created</code> property.
+the same formatting rules as the <code>created</code> property. The updated property
```
Comment (jricher) [inline index.html:3861]:
```suggestion
the same formatting rules as the <code>created</code> property. The <code>updated</code> property
```

**Review by brentzundel — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (shigeya) [reply index.html]: Thanks for spotting this.

File: index.html [hunk 3]
```diff
@@ -3858,7 +3858,11 @@ <h4>updated</h4>
 DID document metadata SHOULD include an <code>updated</code> property to
 indicate the timestamp of the last <a href="#update">Update operation</a> for
 the document version which was resolved. The value of the property MUST follow
-the same formatting rules as the <code>created</code> property.
+the same formatting rules as the <code>created</code> property. The <code>updated</code> property
+is omitted if an Update operation has never been performed on the DID document.
+If an <code>updated</code> property exists, it can be the same value as the
+<code>created</code> property when the difference between the two timestamps
+is less than one second.
```
Comment (TallTed) [inline index.html:3865]: Easier to do this now than later... Just whitespace.
```suggestion
the same formatting rules as the <code>created</code> property. The
<code>updated</code> property is omitted if an Update operation has never been
performed on the DID document. If an <code>updated</code> property exists, it
can be the same value as the <code>created</code> property when the difference
between the two timestamps is less than one second.
```

Comment (shigeya) [reply index.html]: Thanks:)

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**